lighthouse-core/audits/predictive-perf.js

```diff
@@ -7,8 +7,9 @@
 
 const Audit = require('./audit');
 const Util = require('../report/v2/renderer/util.js');
-const PageDependencyGraph = require('../gather/computed/page-dependency-graph.js');
-const Node = require('../gather/computed/dependency-graph/node.js');
+const Simulator = require('../lib/dependency-graph/simulator/simulator.js');
```

Comment:
I guess `simulator.js`'s context can be figured out from its path, but maybe `LoadSimulator` or something for the local handle to it?

Reply:
sg done